Makefile

```diff
@@ -222,16 +222,32 @@ acceptance-test-rpm-package-install: $(SNAPSHOTDIR)
 			$(ACC_TEST_IMAGE) \
 			$(RESULTSDIR)
 
+.PHONY: changlog
+changelog:
+	mkdir -p $(DISTDIR)
+	@docker run -it --rm  \
+		-v "$(shell pwd)":/usr/local/src/your-app ferrarimarco/github-changelog-generator \
```

Comment:
`your-app` ?

Reply:
yup, that's specific to the container they have built, not something we control. The `CHANGELOG.md` will get output to there, so instead of extracting from the container I just mount a volume to get it.

Reply:
But...

`your-app`?